packages/utilities/adaptive-ui/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@microsoft/adaptive-ui",
+  "version": "0.1.0",
```

Comment:
We’re going to minor this into 0.2.0? Should we run this as 1.0.0-alpha or 1.0.0-beta?

Reply:
I don't have an opinion here. Whatever works best with our infrastructure. The only aspect I do have thoughts around is that we are able to use it as a dependency in real packages starting immediately. This represents existing code that's being moved around and repackaged a bit, but the ideas and implementations are stable-enough for v1. We will likely make a bit of an update, particularly around the design token structure, as part of the v2 work, but I don't want to give the impression that that v1 "is not ready".

Reply:
As part of that, are we going to *remove* the parallel exports from fast-foundation?

Reply:
I've updated the version number, but @nicholasrice I wanted to confirm on what you were referring to about the parallel exports. Do you mean the Design Token bit? There shouldn't be anything in here that duplicates or replaces any of that from fast-foundation.

Reply:
Oops nevermind, I was conflating packages and thinking foundation was exporting a lot of this stuff, but it was fast-components which is no removed. Please disregard!